setup.rst

```diff
@@ -106,24 +106,20 @@ Basing your Project on a Specific Symfony Version
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In case your project needs to be based on a specific Symfony version, use the
-optional second argument of the ``new`` command:
+``version`` option of the ``new`` command:
 
 .. code-block:: terminal
 
     # use the most recent version in any Symfony branch
-    $ symfony new my_project_name 2.8
-    $ symfony new my_project_name 3.1
+    $ symfony new --version=3.3 my_project_name
+    $ symfony new --version=3.4 my_project_name
 
     # use a specific Symfony version
-    $ symfony new my_project_name 2.8.3
-    $ symfony new my_project_name 3.1.5
-
-    # use a beta or RC version (useful for testing new Symfony versions)
-    $ symfony new my_project 2.7.0-BETA1
```

Comment:
BETA and RC does not work, I get `Could not find package symfony/skeleton with version 3.4.0-RC1.`.

Event with more recent branches : `Could not find package symfony/skeleton with version 4.3.0-BETA1.` I guess we need to tag the skeleton too?

Reply:
The special version values are `lts`, `stable`, `next`, or `previous`.

We already document `lts`. I'm not sure about `stable` because it's the one used when you don't specify the version, so better not complicate things. About `previous`, I'm not sure either about documenting it in the docs. Finally, the `next` value may be interesting because it allows you to early test whatever the next Symfony version will be.